webapp/client/src/pages/PersonBHasDisabilityPage.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import FormFieldRadio from "../components/FormFieldRadio";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import Details from "../components/Details";
+import { fieldPropType } from "../lib/propTypes";
+
+export default function PersonBHasDisabilityPage({
+  form,
+  fields,
+  prevUrl,
+  stepHeader,
+}) {
+  const { t } = useTranslation();
+
+  const tbold = function (key) {
+    return <Trans t={t} i18nKey={key} components={{ bold: <b /> }} />;
+  };
```

Comment:
I have the feeling we're gonna need this more often. So should we maybe place this in a common place under /lib or something?

Reply:
I can't... I'm not smart enough :confused:. If I move useTranslation() into a seperate js file I got the following error at runtime:

"React Hook "useTranslation" is called in function "translationBold" which is neither a React function component or a custom React Hook function react-hooks/rules-of-hooks"